index.html

```diff
@@ -165,20 +257,120 @@ <h4>Skills</h4>
+            <h4>Language</h4>
+            </div>
+            <div class="ul-karabati">
+            <ul>
+            <li>Javascript</li>
+            <li>Ruby</li>
+            <li>Html</li>
+            <li>Css</li>
+            </ul>
+            </div>
+        </div>
+
+        <div>
+            <div class="karabati">
+                <img src="images/square.svg" alt="about-i" class="about-i-mediacss">
+                <h4>Frameworks</h4>
+            </div>
+            <div class="ul-karabati">
+                <ul>
+                    <li>Bootstrap</li>
+                    <li>Ruby on Rails</li>
+                    <li>RSPec</li>
+                    <li>CapyBara</li>
+                    <li>Selenium</li>
+                    </ul>
+            </div>
+        </div>
+        <div>
+            <div class="karabati">
+                <img src="images/circle.svg" alt="about-i" class="about-i-mediacss">
+                <h4>Skills</h4>
+            </div>
+            <div class="ul-karabati">
+            <ul>
+            <li>Codekit</li>
+            <li>Github</li>
+            <li>Codepen</li>
+            <li>Gitlab</li>
+            <li>Terminal</li>
+            </ul>
+        </div>
+        </div>
+        </div>
+    </section>
+    <!-- For media css -->
+    <div class="mediass-contact">
+        <div>
+        <h3>
+            I'm always interested in hearing about new projects, so if you'd 
+            like to chat please get in touch.
+        </h3>
+        </div>
+        <div>
+        <form action="https://formspree.io/f/xdobraqv" method="post" class="form1-mediacss">
+            <input name="input1" type="text" required class="name" placeholder=" Full name" maxlength="30">
+            <div class="div-intermediate1"></div>
+            <input name="input2" type="email" required class="email" placeholder="Email address">
+            <div class="div-intermediate2"></div>
+            <textarea name="text-area" required id="her" cols="30" rows="10" class="info-media" placeholder="Hello Mariane. Glad to see you |" maxlength="500"></textarea>
+            <button class="button-media" type="button">Get in touch</button>
+        </form>
+    </div>
+    </div>
+    <div class="div-low-for-med"></div>
+        
+    <ul class="social-for-med">
+        <li class="s1"><img src="images/vfooter.png" alt="GitHub"></li>
+        <li class="s2"><img src="images/link.png" alt="linkedin"></li>
+        <li class="s3"><img src="images/chill.png" alt="handsup"></li>
+        <li class="s4"><img src="images/twitter.png" alt="twitter"></li>
+        <li class="s5"><img src="images/em.png" alt="Media"></li>
+    </ul>
+
+    
+    
+    <!-- For mobile screen -->
     <section class="content">
         <h3>
             I'm always interested in hearing about new projects, so if you'd 
             like to chat please get in touch.
         </h3>
     </section>
```

Comment:
### Duplicate Paragraph
![duplicate-intro](https://user-images.githubusercontent.com/93039359/177857322-48da2dfc-58be-440d-b465-6605384285f1.png)

- [ ] Your mobile version is showing duplicate paragraphs. Please remove one of them :+1: